CHANGELOG.md

```diff
@@ -1,15 +1,13 @@
-# NEXT RELEASE
+# 14.6.0 Release notes
```

Comment:
should you add the section for next release now?

Reply:
I wasn't sure if it was added as part of this merge or in a separate PR. I can add it now.

Reply:
don't know either. but at least now we update the changelog for the next release